src/init/application/templates/polymer-2.x/test/_element/_element_test.html

```diff
@@ -12,19 +12,42 @@
     <link rel="import" href="../../src/<%= elementName %>/<%= elementName %>.html">
   </head>
   <body>
-    <test-fixture id="basic">
+
+    <test-fixture id="BasicTestFixture">
```

Comment:
test-fixture documentation uses UpperCamelCase everywhere (see https://github.com/PolymerElements/test-fixture README, jsdoc, etc.). I want us to be a good example for new users and not introduce confusion by breaking from those docs.